### DISCUSSION TITLE
How to select EDM sampler?

### DISCUSSION THREAD

**Discussion by SAC020** [vladmandic/sdnext#2950] [General]
Changelog mentions EDM samplers (1-Mar-2024 update). I cannot figure out how to select or activate EDM, there is no such sampler listed and searching EDM in settings doesn't return anything

Didn't open this as an issue as it's most likely a setting that I am missing, and not a bug.

Thx

```
16:59:50-289891 INFO     Starting SD.Next
16:59:50-293286 INFO     Logger: file="C:\ai\automatic\sdnext.log" level=DEBUG size=65 mode=create
16:59:50-294774 INFO     Python 3.10.11 on Windows
16:59:50-466992 INFO     Version: app=sd.next updated=2024-03-05 hash=76b00176
                         url=https://github.com/vladmandic/automatic/tree/dev
16:59:50-881007 INFO     Latest published version: 912237ecf7d5b3616a272f983f3f59cc405f64c3 2024-03-01T14:24:24Z
16:59:50-894399 INFO     Platform: arch=AMD64 cpu=Intel64 Family 6 Model 165 Stepping 5, GenuineIntel system=Windows
                         release=Windows-10-10.0.22631-SP0 python=3.10.11
```

![image](https://github.com/vladmandic/automatic/assets/50420272/f511c20e-7679-461f-ae4d-de4837b44edf)

![image](https://github.com/vladmandic/automatic/assets/50420272/1e8ec62a-9116-4a1d-aa02-242462a7a525)

**Comment by vladmandic** [comment]
changelog also says that EDM samplers are only available in diffusers==0.27.0.dev, not in normal released version.

**Reply by SAC020** [reply]
But I am on dev, and updated today. Do I need to do / set something else?

```
16:59:50-466992 INFO     Version: app=sd.next updated=2024-03-05 hash=76b00176
                         url=https://github.com/vladmandic/automatic/tree/dev
```

**Reply by vladmandic** [reply]
diffusers==0.27.0.dev in addition to sdnext dev!
samples are not in released version of diffusers
